### PR TITLE
docs: revert logo

### PR DESCRIPTION
**Pull request by benjamincanac**

### CODE REVIEW

File: docs/pages/[...slug].vue [hunk 1]
```diff
@@ -20,9 +20,9 @@ const { data: surround } = await useAsyncData(`${route.path}-surround`, () => qu
 )
 
 useSeoMeta({
-  titleTemplate: '%s - Tailwind CSS module for Nuxt',
+  titleTemplate: '%s - Nuxt Tailwind',
```
Comment (ineshbose) [inline docs/pages/[...slug].vue:23]: I think "Nuxt Tailwind" is forbidden by Tailwind CSS trademark - https://tailwindcss.com/brand#trademark-usage-agreement

File: docs/content/index.yml
Comment (ineshbose) [inline docs/content/index.yml:1]: Changes to this file can be kept as most of it was language improvement (in my opinion)

File: docs/content/2.tailwind/2.viewer.md
Comment (ineshbose) [inline docs/content/2.tailwind/2.viewer.md:1]: Changes can be kept as it is purely language improvement

File: docs/components/AppHeader.vue [hunk 2]
```diff
@@ -7,11 +7,9 @@ const { header } = useAppConfig()
 </script>
 
 <template>
-  <UHeader :ui="{ logo: 'items-center' }">
+  <UHeader>
     <template #logo>
-      <TheLogo class="h-12" />
-      <span>Nuxt</span>
-      <TailwindText class="h-3.5" />
```
Comment (ineshbose) [inline docs/components/AppHeader.vue:14]: this keeps the "tailwindcss" text aside from Nuxt text which is appropriate I think

File: docs/pages/[...slug].vue (hunk 1 above)
Comment (benjamincanac) [reply docs/pages/[...slug].vue:23]: This is how we've branded all our modules so I'll let @Atinux change it back if needed.

File: docs/components/AppHeader.vue (hunk 2 above)
Comment (benjamincanac) [reply docs/components/AppHeader.vue:14]: `Nuxt` is already in the logo

File: docs/pages/[...slug].vue (hunk 1 above)
Comment (ineshbose) [reply docs/pages/[...slug].vue:23]: While @Atinux also gets to this comment, I'll link the discussion from the insiders meet too - https://github.com/nuxt/insiders/discussions/69#discussioncomment-6637547 where I mention about the third-party libraries' guidelines. The module naming would need to be compliant, along with the logo being compliant (either not having a logo or using Nuxt and Tailwind CSS logos side-by-side). To keep the logo in the square form factor, I thought of putting the winds through the mountains but not 100% sure if that works or not (execution isn't great since I'm not a graphic designer), but lets really clarify this (about the naming and logo usage) please.

File: docs/pages/index.vue
Comment (ineshbose) [inline docs/pages/index.vue:1]: Why this change @benjamincanac?

File: docs/components/AppHeader.vue (hunk 2 above)
Comment (ineshbose) [reply docs/components/AppHeader.vue:14]: Yes, but the 'Tailwind' brand isn't appropriate I believe - lets continue in https://github.com/nuxt-modules/tailwindcss/pull/873#discussion_r1673711472

File: docs/pages/index.vue
Comment (benjamincanac) [reply docs/pages/index.vue:1]: This is how it's done on all the docs, that's all. Doesn't change anything otherwise.

Comment (benjamincanac) [reply docs/pages/index.vue:1]: I reverted the change

**Review by ineshbose — APPROVED** [review]
Will follow up on the discussions when @Atinux is available. Approving for now! 👍
